RecoTracker/SpecialSeedGenerators/src/SeedGeneratorForCRack.cc

```diff
@@ -18,11 +19,14 @@ void SeedGeneratorForCRack::init(const SiStripRecHit2DCollection& collstereo,
 
   iSetup.get<TransientRecHitRecord>().get(builderName, theBuilder);
   TTTRHBuilder = theBuilder.product();
-  CosmicLayerPairs cosmiclayers(geometry);
-  cosmiclayers.init(collstereo, collrphi, collmatched, iSetup);
-  thePairGenerator = new CosmicHitPairGenerator(cosmiclayers, iSetup);
+  edm::ESHandle<GeometricSearchTracker> track;
+  iSetup.get<TrackerRecoGeometryRecord>().get(track);
+  edm::ESHandle<TrackerTopology> httopo;
+  iSetup.get<TrackerTopologyRcd>().get(httopo);
```

Comment:
is this and 4 more `es::get` left for another PR?

Reply:
That class apparently isn't used in any of the RelVals so its conversion isn't pressing.

Reply:
This is occasionally used by the SiStrip group to test reconstruction with the Cosmic Rack, I can have a look next week.

Reply:
See https://github.com/cms-sw/cmssw/pull/35367